react-native.config.js

```diff
@@ -4,4 +4,13 @@ module.exports = {
         android: {},
     },
     assets: ['./assets/fonts/native'],
+    expo: {
+        userInterfaceStyle: 'automatic',
+        ios: {
+            userInterfaceStyle: 'automatic',
+        },
+        android: {
+            userInterfaceStyle: 'automatic',
+        },
+    },
```

Comment:
Is this necessary? App is not using expo yet

Reply:
no you're right, I can remove and rebuild